deployment/plat-app-services/static-page/manifest.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: static-page
+  name: static-page
+spec:
+  ports:
+  - name: http
+    nodePort: 30000
+    port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: static-page
+  type: NodePort
```

Comment:
We should avoid node ports if possible. But it just occurred to me this file isn;t actually included in the Kustomize build, so I guess you were just testing?

Reply:
Yes apologies that should have been removed, will just confirm.

Reply:
The file manifest.yaml should not have been included it was from an earlier attempt at this. It has been removed now.